### PR TITLE
docs(tutorials): add Pulumi quickstart tutorial

### PR DESCRIPTION
**Pull request by dirien**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Hi,

created a quickstart tutorial for Pulumi. In this ttutorial, we learn how to install pulumi and create / modify and delete a Scaleway Kapsule.

Looking for your feedback.

### CODE REVIEW

File: tutorials/pulumi-quickstart/index.mdx [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+  aws-python                   A minimal AWS Python Pulumi program
+  aws-typescript               A minimal AWS TypeScript Pulumi program
+  azure-csharp                 A minimal Azure Native C# Pulumi program
+  azure-go                     A minimal Azure Native Go Pulumi program
+  azure-javascript             A minimal JavaScript Pulumi program with the native Azure provider
+  azure-python                   A minimal Azure Native Python Pulumi program
+  ....
+  ```
+
+3. Add the Scaleway provide`r`
```
Comment (bene2k1) [inline tutorials/pulumi-quickstart/index.mdx:166]:
```suggestion
3. Add the Scaleway provider
```

File: tutorials/pulumi-quickstart/index.mdx [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+
+```
+$ export KUBECONFIG=kubeconfig.yaml && kubectl get nodes
+NAME                                             STATUS   ROLES    AGE     VERSION
+scw-pulumi-kapsu-pulumi-kapsule-pool-sm-059c03   Ready    <none>   3m56s   v1.23.6
+scw-pulumi-kapsu-pulumi-kapsule-pool-sm-905ac2   Ready    <none>   4m18s   v1.23.6
+scw-pulumi-kapsule-pulumi-kapsule-pool-6d0c822   Ready    <none>   76m     v1.23.6
+```
+
+## Delete your infrastructure
```
Comment (bene2k1) [inline tutorials/pulumi-quickstart/index.mdx:414]:
```suggestion
## Deleteing your infrastructure
```

File: tutorials/pulumi-quickstart/index.mdx (hunk 1 above)
Comment (dirien) [reply tutorials/pulumi-quickstart/index.mdx:163]: Ups! 😅

**Comment by dirien** [comment]
Hi @bene2k1,

thanks for reviewing the PR and the suggestions to improve it! Commited all of them! ❤️

**Comment by dirien** [comment]
Should I squash all commits or are you doing this when approving the PR, @bene2k1?

File: tutorials/pulumi-quickstart/index.mdx [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+<Message type="tip">
+
+  - You can skip the preview by using the flag `--skip-preview` or `-f`
+  - You can skip the confirmation dialog by using `--yes` or `-y`
+
+  Both flags are very useful, when you run Pulumi in a CI environment.
+
+</Message>
+
+  The update process will take some minutes, and you should be seeing following output when everything is done:
```
Comment (nedaehm) [inline tutorials/pulumi-quickstart/index.mdx:282]:
```suggestion
  The update process will take a few minutes, and you should see the following output when everything is done:
```

File: tutorials/pulumi-quickstart/index.mdx [hunk 4]
```diff
@@ -0,0 +1,488 @@
...
+
+```
+$ export KUBECONFIG=kubeconfig.yaml && kubectl get nodes
+NAME                                             STATUS   ROLES    AGE     VERSION
+scw-pulumi-kapsu-pulumi-kapsule-pool-sm-059c03   Ready    <none>   3m56s   v1.23.6
+scw-pulumi-kapsu-pulumi-kapsule-pool-sm-905ac2   Ready    <none>   4m18s   v1.23.6
+scw-pulumi-kapsule-pulumi-kapsule-pool-6d0c822   Ready    <none>   76m     v1.23.6
+```
+
+## Deleteing your infrastructure
```
Comment (nedaehm) [inline tutorials/pulumi-quickstart/index.mdx:411]:
```suggestion
## Deleting your infrastructure
```

**Comment by bene2k1** [comment]
@dirien please stash your PR before it is going to be merged

**Comment by dirien** [comment]
Hi @bene2k1,

done :)

File: tutorials/pulumi-quickstart/index.mdx [hunk 5]
```diff
@@ -0,0 +1,485 @@
...
+Resources:
+  - 4 deleted
+
+Duration: 59s
+
+The resources in the stack have been deleted, but the history and configuration associated with the stack are still maintained.
+If you want to remove the stack completely, run 'pulumi stack rm dev'.
+```
+
+Bravo, that's it! You got a first impression on the lifecycle of an infrastructure using Pulumi. There is so much more to exiting aspects to discover, feel free to jump to the official Pulumi [docs](https://www.pulumi.com/docs/).
```
Comment (nedaehm) [inline tutorials/pulumi-quickstart/index.mdx:485]:
```suggestion
You got a first impression on the lifecycle of an infrastructure using Pulumi. There are many more aspects to discover. Check the official Pulumi [docs](https://www.pulumi.com/docs/) to find out more.
```

**Comment by nedaehm** [comment]
Hello @dirien ! Thanks for contributing to Scaleway Documentation! Once everything will be reviewed, we will merge your tutorial and publish it on social media! We just added a 200euros discount coupon (valid until December 2022 included) to your Scaleway account. Do not hesitate to contribute again 🙂

**Comment by dirien** [comment]
Thanks @nedaehm, looking forward to contribute more!

**Review by bene2k1 — APPROVED** [review]

**Review by nedaehm — APPROVED** [review]

**Pull request merged**